### ISSUE TITLE
merge after readonly  the view update Untimely

### ISSUE THREAD

**Issue by rianran1993** [handsontable/handsontable#5184]
### Description

merge after readonly  the view update Untimely
### Steps to reproduce

1. select > 2 cells to be readonly
2. set the cells you selceted step 1 to merge
3. then you will see the cells not merged immediately

### Demo

https://handsontable.com/examples.html?headers&merge-cells

### Your environment
* Handsontable version: official demo
* Browser Name and version: official demo
* Operating System: official demo

**Comment by AMBudnik** [comment]
Thank you for sharing @rianran1993 

Here's what I got

![gif](https://user-images.githubusercontent.com/10757813/41454769-fc1b273c-707a-11e8-845d-3d4a58862a3e.gif)

If a cell is readOnly we cannot change its value and as merging changes the value of a cell this case it tricky. However, the selection gets messy which is definitely a bug.

**Comment by AMBudnik** [comment]
We've just discussed the subject and decided to blur the button for `merge cells` while readOnly cell is a part of the selected area.

**Comment by rianran1993** [comment]
ok,  You can try another step ,you will see the view will update.

1. select > 2 cells to be readonly
2. set the cells you selceted step 1 to merge
3. change a cell's value which not in the selection, you will see the merge area effect

**Comment by AMBudnik** [comment]
We can create a new issue for this second scenario.

**Comment by rianran1993** [comment]
ok, maybe you can create a new issue directly. In this situation, I have no choice but set the cells to be writable if I want to use merge,right?

**Comment by AMBudnik** [comment]
Yes, I will. We're having a meeting within 10 minutes so after that, I will test it a little bit more and post a new issue.

**Comment by AMBudnik** [comment]
You may try to set `editor: false` instead of `readOnly` but it is not a part of a default context menu set of options so you'd need to create a custom menu.
I think taht we can keep the second scenario here until some developer will take the case.

**Comment by rianran1993** [comment]
ok, I'll try.  thanks

**Comment by AMBudnik** [comment]
Also replicable with 7.2.1 and minimal settings https://jsfiddle.net/gkqh3odj/

**Comment by AMBudnik** [comment]
Works the same in beta1 of 8.0.0 https://jsfiddle.net/dqLze1cs/

**Comment by sl01k** [comment]
Tested on Handsontable v17 — merging readOnly cells now updates the viewimmediately. The merge is visually applied right away without needing another action to trigger a re-render.

Demo: https://jsfiddle.net/oL0evtwp/1/

Closing as resolved.
